src/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Regex.Cache.cs

```diff
@@ -138,6 +163,8 @@ public override int GetHashCode()
         /// </summary>
         internal sealed class CachedCodeEntry
         {
+            public CachedCodeEntry _next = null;
+            public CachedCodeEntry _previous = null;
             public CachedCodeEntryKey _key;
```

Comment:
Can we make `_key` `readonly` since only the ctor writes to it?
Actually all the other fields, except _next and _previous, look like they can be `readonly`.

Reply:
Really they should be properties, but that's just style and doesn't need to change now.

Reply:
All instance fields here except _factory and _code should be readonly and and uppercase i.e. Next instead of _next.

Reply:
Ok. There is also the same in Regex (_code and other).

Reply:
Oh, no, they are private.